axelrod/tests/strategies/test_stalker.py

```diff
@@ -21,26 +21,39 @@ class TestStalker(TestPlayer):
     }
 
     def test_strategy(self):
+        
         # Start with cooperation
         self.first_play_test(C)
 
         # current_average_score > very_good_score
-        self.responses_test([D], [C] * 2 + [D] * 4, [D] * 2 + [C] * 4)
```

Comment:
There are two tests here but one replacement. I think if we transcribe the tests exactly we should have the coverage we had before.

This line:
```
self.responses_test([D], [C] * 2 + [D] * 4, [D] * 2 + [C] * 4)
```

corresponds to

```
actions = [(C, D)] * 2 + [(D, C)] * 4 + [(D, *)]
```
where * indicates a missing value from the original test (since the original test only looked at the player's next move and not the opponent's).